src/main/java/com/bytezone/dm3270/display/ScreenPacker.java

```diff
@@ -27,24 +25,25 @@ public Command readModifiedFields(byte currentAID, int cursorLocation,
       boolean readModifiedAll, boolean sscpLuData) {
     // pack the AID
     int ptr = 0;
+
     if (!sscpLuData) {
```

Comment:
Move this if after the following one to avoid unnecessary creation of BufferAddress and packing when PAX  and CLEAR attention keys and incorrect pointer address when creating such attention keys.